components/textfield/metadata/textarea.yml

```diff
@@ -1,6 +1,6 @@
-name: Text area
+name: Text Area
```

Comment:
This just makes things match the rest of the project. Jess reports that it needs to be sentence case, so we can fix that for everything in #463.